zcutil/fetch-params.py

```diff
@@ -0,0 +1,329 @@
+# /************************************************************************
+#  File: fetch-params.py
+#  Author: mdr0id
+#  Date: 1/18/2019
+#  Description:  Used to distribute ceremony parameters, per platform, 
+#                for zcashd
+#
+#  Usage: python3 fetch-params.py
+#
+#         OR USING IPFS:
+#
+#         python3 fetch-params.py --ipfs
+#         
+#         To generate a binary for a given platform using pyinstaller module :
+#
+#         pip3 install pyinstaller
+#
+#         pyinstaller --onefile fetch-params.py
+#
+#         <binary will be located in /dist of current working directory>
+#
+#
+# Known Bugs:
+#
+# - IPFS Daemon: ERROR core: failure on stop:  context canceled; TODO: mock repo builder.go:47
+#   see https://github.com/ipfs/go-ipfs/issues/6356 for details
+#
+# ************************************************************************/
+import sys
+import platform
+import hashlib
+import os
+import argparse
+from subprocess import Popen, PIPE
+
+try:
+    from logzero import logger
+except ImportError as error:
+    print(error.__class__.__name__, " : ", error)
+    print(sys.version)
```

Comment:
If any of these import attempts fail, we should exit with return code `1` since they're fatal.

```
sys.exit(1)
```